include/hal/library/requester/reqasymsignlib.h

```diff
@@ -65,6 +67,7 @@ extern bool libspdm_encap_challenge_opaque_data(
     void *spdm_context,
     spdm_version_number_t spdm_version,
     uint8_t slot_id,
+    uint64_t request_context,
```

Comment:
how about to use `uint8_t *request_context`?

Just in case that, the request_context is enlarged (more than 8 bytes) in next version.